fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -3100,8 +3102,13 @@ public void addPartitions(Database db, String tableName, AddPartitionClause addP
 
             Column firstPartitionColumn = partitionColumns.get(0);
             MultiRangePartitionDesc multiRangePartitionDesc = (MultiRangePartitionDesc) partitionDesc;
+            Map<String, String> properties = addPartitionClause.getProperties();
```

Comment:
Is it possible that properties is null?

Reply:
Usually it won’t be null, but better add a check.